storage/api/Storage.Samples.Tests/SetRPODefaultTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿
+// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+[Collection(nameof(StorageFixture))]
+public class SetRPODefaultTest
+{
+    private readonly StorageFixture _fixture;
+
+    public SetRPODefaultTest(StorageFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    [Fact]
+    public void SetRPODefault()
+    {
+        SetRPODefaultSample setRPODefaultSample = new SetRPODefaultSample();
+        GetBucketMetadataSample getBucketMetadataSample = new GetBucketMetadataSample();
+        
+        setRPODefaultSample.SetRPODefault(_fixture.BucketNameGeneric);
```

Comment:
Should you create a new bucket for this test? I don't remember what the other tests that change the metadata do, but maybe we shouldn't change the metadata for the generic bucket as it might make it useless for tests that come later.

Reply:
I could see these tests creating fresh temporary buckets in the instance for each test run. Am I missing something?